lib/commands/app-management.js

```diff
@@ -17,13 +17,19 @@ export const APP_STATE = /** @type {const}  */ ({
   RUNNING_IN_FOREGROUND: 4,
 });
 
+/**
+ * @typedef {Object} IsAppInstalledOptions
+ * @property {string} [user] - The user id
+ */
+
 /**
  * @this {AndroidDriver}
  * @param {string} appId
+ * @param {IsAppInstalledOptions} [opts={}]
  * @returns {Promise<boolean>}
  */
-export async function isAppInstalled(appId) {
-  return await this.adb.isAppInstalled(appId);
+export async function isAppInstalled(appId, opts) {
```

Comment:
should opts have a default value?

Reply:
yea, 2cfe880b23eaea0317c3d70af05eac96de32b812